plugins/modules/system/lvg.py

```diff
@@ -40,6 +40,11 @@
     description:
     - Additional options to pass to C(pvcreate) when creating the volume group.
     type: str
+  pvresize:
+    description:
+    - If C(yes) resize the physical volume to the maximum available size.
+    type: bool
+    default: false
```

Comment:
```suggestion
  pvresize:
    description:
    - If C(yes) resize the physical volume to the maximum available size.
    type: bool
    default: false
```
BTW , can't it be an option of state? Like [absent, present, resize]?

Reply:
I think this would be misleading, because "present" state currently performs resizes (When adding new devices). I also believe that the state should refer to the "main subject" of the module, which in this case are volume groups. Physical volumes are just an "extra" in this module

Reply:
ok